src/ServiceControl/Infrastructure/RavenDB/Expiration/ExpiredDocumentsCleaner.cs

```diff
@@ -12,13 +12,7 @@ class ExpiredDocumentsCleaner
     {
         public static Task<TimerJobExecutionResult> RunCleanup(int deletionBatchSize, DocumentDatabase database, Settings settings, CancellationToken token)
         {
-            var threshold = SystemTime.UtcNow.Add(-settings.AuditRetentionPeriod);
-
-            logger.Debug("Trying to find expired ProcessedMessage and SagaHistory documents to delete (with threshold {0})", threshold.ToString(Default.DateTimeFormatsToWrite, CultureInfo.InvariantCulture));
-            AuditMessageCleaner.Clean(deletionBatchSize, database, threshold, token);
-            SagaHistoryCleaner.Clean(deletionBatchSize, database, threshold, token);
-
-            threshold = SystemTime.UtcNow.Add(-settings.ErrorRetentionPeriod);
+            var threshold = SystemTime.UtcNow.Add(-settings.ErrorRetentionPeriod);
```

Comment:
I know it was there before but what is the purpose of using RavenDB SystemTime vs DateTime.UtcNow?

Reply:
It has been this way since the expiration bundle was first written (Feb 2014). As far as I know, only the saga audit tests use SystemTime.

Reply:
ok, lets keep as is for now?